src/dune_lang/dune_lang.ml

```diff
@@ -6,6 +6,4 @@ module Glob = Glob
 module String_with_vars = String_with_vars
 module Pform = Pform
 module Action = Action
-
-(* TODO remove the [Dune] prefix *)
```

Comment:
@rgrinberg FTR I was following the instruction of this comment before. Now with `Dune_file_prefix`, removing the `Dune` prefix doesn't make sense since it is about a `Dune_file`. But I guess that's ok.

Reply:
It should have said replace the dune prefix with something more descriptive I suppose. `Dune_lang.Lexer` alone doesn't really tell us what is being lexed.